safor/fonts.css

```diff
@@ -0,0 +1,30 @@
+@font-face {
```

Comment:
You have 5 fonts declared here, however, there are 6 fonts in the fonts folder, is there a missing font declaration?